README.md

```diff
@@ -20,7 +20,69 @@ Once installed, run the client at the commandline with:
  $ wasapi-client --help
 ```
 
-If you are running this to access the Archive-It WASAPI endpoint,
+That gives you usage instructions:
+
+```
+usage: wasapi-client [-h] [-b BASE_URI] [-d DESTINATION] [-l LOG] [-n]
+                     [-u USER] [-v] [-c | -m | -p PROCESSES | -s | -r]
+                     [--collection COLLECTION [COLLECTION ...]]
+                     [--filename FILENAME] [--crawl CRAWL]
+                     [--crawl-time-after CRAWL_TIME_AFTER]
+                     [--crawl-time-before CRAWL_TIME_BEFORE]
+                     [--crawl-start-after CRAWL_START_AFTER]
+                     [--crawl-start-before CRAWL_START_BEFORE]
+
+        Download WARC files from a WASAPI access point.
+
+        Acceptable date/time formats are:
+         2017-01-01
+         2017-01-01T12:34:56
+         2017-01-01 12:34:56
+         2017-01-01T12:34:56Z
+         2017-01-01 12:34:56-0700
+         2017
+         2017-01
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -b BASE_URI, --base-uri BASE_URI
+                        base URI for WASAPI access; default:
+                        https://partner.archive-it.org/wasapi/v1/webdata
+  -d DESTINATION, --destination DESTINATION
+                        location for storing downloaded files
+  -l LOG, --log LOG     file to which logging should be written
+  -n, --no-manifest     do not generate checksum files (ignored when used in
+                        combination with --manifest
+  -u USER, --user USER  username for API authentication
+  -v, --verbose         log verbosely; -v is INFO, -vv is DEBUG
+  -c, --count           print number of files for download and exit
+  -m, --manifest        generate checksum files only and exit
+  -p PROCESSES, --processes PROCESSES
+                        number of WARC downloading processes
+  -s, --size            print count and total size of files and exit
+  -r, --urls            list URLs for downloadable files only and exit
+
+query parameters:
+  parameters for webdata request
+
+  --collection COLLECTION [COLLECTION ...]
+                        collection identifier
+  --filename FILENAME   exact webdata filename to download
+  --crawl CRAWL         crawl job identifier
+  --crawl-time-after CRAWL_TIME_AFTER
+                        request files with date of creation during a crawl job
+                        after this date
+  --crawl-time-before CRAWL_TIME_BEFORE
+                        request files with date of creation during a crawl job
```

Comment:
This and the previous description seem a bit confusing to me, particularly the "during a crawl job" part. Didn't notice it in the previous PR, but here I think it could be worded more clearly, like the subsequent descriptions.

Reply:
That is confusing, I don't even know what that is supposed to mean.